aws-rds/aws-rds.json

```diff
@@ -1767,7 +1724,7 @@
             "filter": ""
           },
           "matchExact": true,
-          "metricName": "OldestReplicationSlotLag",
```

Comment:
I would prefer to keep original metric here. ReplicationSlotDiskUsage can be additional panel, query.

Reply:
There's another panel with the original metric called `OldestReplicationSlotLag/ReplicaLag` this one's a duplicate panel which has title `ReplicationSlotDiskUsage/TransactionLogsDiskUsage`